packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -5,22 +5,91 @@ const cli = {
   describe: 'Run wdio tests',
   builder: (yargs) => (
     yargs.options({
+      assetServerPort: {
+        type: 'number',
+        describe: 'The port to run the webpack and express asset services on.',
+        default: () => {
+          if (process.env.WDIO_INTERNAL_PORT) {
+            return process.env.WDIO_INTERNAL_PORT;
+          }
+
+          return 8080;
+        },
+      },
       browsers: {
         type: 'array',
         describe: 'A list of browsers for the test run.',
+        default: () => {
+          let browsers = process.env.BROWSERS;
+          if (browsers) {
+            // Remove the brackets if BROWSERS is set as an array like ['chrome, firefox, ie'] or ['chrome','firefox','ie'] in Jenkinsfile.
+            browsers = browsers.replace('[', '');
+            browsers = browsers.replace(']', '');
+
+            // Split the list of BROWSERS into an array if it is set in the string form like 'chrome,firefox,ie'.
+            if (browsers.includes(',')) {
+              return browsers.split(',');
+            }
+
+            return [browsers];
+          }
+
+          return [];
+        },
       },
       c: {
         type: 'string',
         alias: 'config',
```

Comment:
should the alias and object key be switched? it looks like what we have defined results in `--c, -config`

Reply:
That is what I thought at first. But basically yargs sets the key names as equivalent such that updates to a key will propagate to aliases and vice-versa.

Optionally .alias() can take an object that maps keys to aliases. Each key of this object should be the canonical version of the option, and each value should be a string or an array of strings.

This is their documentation: https://yargs.js.org/docs/#api-reference-optionskey-opt